usagov/vote-gov-drupal
# VOTE-2777 add alternate route for Spanish NVRF

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2777

## Description

Add an alternative route for Spanish NVRF

## Deployment and testing

### Post-deploy steps

1. `lando retune`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/admin/content/block/3 and add content in the "Digital Form" fields
2. edit the spanish translation http://vote-gov.lndo.site/es/admin/content/block/3 and add content in the "Digital Form" fields
3. visit http://vote-gov.lndo.site/register/minnesota and click on the form filler link and make sure it renders the NVRF app
4. click on the "Español" button and make sure it goes to `/es/registrar/minnesota/mail-in-form-filler`
5. click on the "back to registration options" link and make sure it goes back the minnesota page in spanish.
6. click on the form filler link and make sure it goes to the spanish NVRF path `/es/registrar/minnesota/mail-in-form-filler`

## Checklist for the Developer




## Checklist for the Peer Reviewers
